template/build/webpack.dev.conf.js

```diff
@@ -31,6 +31,7 @@ module.exports = merge(baseWebpackConfig, {
       filename: 'index.html',
       template: 'index.html',
       inject: true,
+      subDirectory: config.dev.assetsSubDirectory,
```

Comment:
I can't find this option in that plugin's documentation. Do you have a link or something?